packages/grafana-data/src/types/datasource.ts

```diff
@@ -392,6 +405,11 @@ export interface DataQuery {
    */
   queryType?: string;
 
+  /**
+   * Specify the explict channel for where the query results should be sent
+   */
+  queryChannel?: DataQueryChannel;
```

Comment:
Not sure a good term to use here -- any ideas?

I would like to reserve "stream" and "channel" for the "live" streaming feature.

Maybe:
* DataQuerySubject?
* DataQueryTopic?
* DataQuery????

Reply:
Think `topic` is a good comporomise!